packages/ez_file/ez_file.0.1.2/opam

```diff
@@ -0,0 +1,44 @@
+# This file was generated by `drom` from `drom.toml`.
+# Do not modify or add to the `skip` field of `drom.toml`.
+opam-version: "2.0"
+name: "ez_file"
```

Comment:
Can you please remove name and version? They are inferred from the opam file path in any case

Reply:
What is the problem with having them in the `opam` file ? It's quite useful to have a file which is self-contained, instead of relying on the fact that it is in a directory NAME.VERSION.

Reply:
No particular reason, mostly cleaning redundant information. In the past we had a few cases in which versions between opam files, opam metadata and dune did not match and created warnings all over the place